frontend/src/components/UI/PaginationNav/PaginationNav.module.scss

```diff
@@ -0,0 +1,3 @@
+.hideElement {
```

Comment:
we may have a this css class in our css globals

Reply:
sorry was thinking about `display-none` from [uswds](https://designsystem.digital.gov/utilities/display/#display-2)

Reply:
didn't see it anyplace